setup.py

```diff
@@ -30,46 +29,29 @@
     'tutorial': ['jupyter'],
 }
 
-
-def main():
-    pack_dir = {
-        'ixmp': 'ixmp',
-    }
-    entry_points = {
+LIB_FILES = [x.split('ixmp/')[-1] for x in glob.glob('ixmp/lib/*')]
+
+setup(
+    name='ixmp',
+    version=versioneer.get_version(),
+    description='ix modeling platform',
+    author='Daniel Huppmann, Matthew Gidden, Volker Krey, '
```

Comment:
actually here, can we just `cat` the contents of `Authors.md` (or refer to it)? In other projects I work on, if you author a commit, you get to be on the author list. Some are a bit more strict (keeping lists per release), but I think that is too much overhead for us..

Reply:
There's an AUTHORS.md in message_ix but none in ixmp. See also https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py#10005265 —which is why I suggest using "IIASA Energy Program [and contributors]" in packaging/the copyright notice, and then having the full list in a file/the docs.